File: LinkERP/ClientApp/src/app/models/inv/lbs-inv-productstylematrixdetail.ts
import { BaseEntity } from "../base-entity";
import { LBS_INV_ProductStyleMatrix } from "./LBS_INV_ProductStyleMatrix";
export class LbsInvProductstylematrixdetail extends BaseEntity{
    CompanyID:any;
    StyleMatrixID:any;
    StyleMatrixDetailCode:any;
    StyleMatrixDetailName:any;
    Product: any = new LBS_INV_ProductStyleMatrix();
}

File: LinkERP.Entity/INV/LBS_INV_ProductStyleMatrixDetail.cs
using LinkERP.Entity.SYS;
using System;
using System.Collections.Generic;
using System.Text;

namespace LinkERP.Entity.INV
{
  public  class LBS_INV_ProductStyleMatrixDetail :BaseEntity
    {
        public Guid CompanyID { get; set; }
        public Guid StyleMatrixID { get; set; }
        public string StyleMatrixDetailCode { get; set; }
        public string StyleMatrixDetailName { get; set; }
        public LBS_SYS_Company LBS_SYS_Company { get; set; }
        public LBS_INV_ProductStyleMatrix LBS_INV_ProductStyleMatrix { get; set; } 
      
    }
}
